src/updateChangelog.ts

```diff
@@ -156,10 +171,11 @@ async function updateChangelog({
 
   const loggedPrNumbers = getAllLoggedPrNumbers(changelog);
   const newCommits = commits.filter(
-    ({ prNumber }) => !loggedPrNumbers.includes(prNumber),
+    ({ prNumber }) => !loggedPrNumbers.includes(prNumber as any),
```

Comment:
Instead of using `as any` here, we could satisfy `tsc` by adding `prNumber === undefined || ` to the front of this expression

Reply:
```suggestion
    ({ prNumber }) => prNumber === undefined || !loggedPrNumbers.includes(prNumber),
```